app/src/main/java/com/hedvig/app/feature/offer/PerilBottomSheet.kt

```diff
@@ -0,0 +1,68 @@
+package com.hedvig.app.feature.offer
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.hedvig.android.owldroid.graphql.OfferQuery
+import com.hedvig.app.BuildConfig
+import com.hedvig.app.R
+import com.hedvig.app.util.extensions.isDarkThemeActive
+import com.hedvig.app.util.svg.buildRequestBuilder
+import e
+import kotlinx.android.synthetic.main.peril_bottom_sheet_new.*
+
+class PerilBottomSheet : BottomSheetDialogFragment() {
```

Comment:
Does this sheet differ from the one in `ContractCoverageActivity` at all? 🙂 If not, we can simply use that one, no need to maintain two files 🙂